src/main.ts

```diff
@@ -129,19 +134,32 @@ async function compile() {
       bundle(files.Ok).then((bundled) => {
         const evalResult = eval(bundled);
 
+        logger.clear();
+        logger.mountGlobally();
+
         if (evalResult != undefined && evalResult.hasOwnProperty("main")) {
-          document.getElementById("eval-output").textContent = evalResult.main();
+          try {
+            logger.log(evalResult.main());
+          } catch (e) {
+            if (e.gleam_error) {
+              logger.error(`Error: ${e.gleam_error}\n  module: ${e.module}\n  line:`, e.line, `\n  fn: ${e.fn}\n  value:`, e.value)
+            } else {
+              logger.error(e);
+            }
+          }
         } else {
-          document.getElementById("eval-output").textContent = "Main function not found. It is defined and public?";
+          logger.log("Main function not found. It is defined and public?");
         }
+        logger.unmountGlobally();
       });
     } else {
       erlangEditor.setValue(files.Ok["build/dev/erlang/gleam-wasm/main.erl"]);
 
       document.getElementById("eval-output").textContent = "Compiled successfully!\n\nNote that the Erlang target is not executable in the browser.";
```

Comment:
I believe this would need to use `logger.log` as well?

Reply:
You are correct. I'll fix it in a moment.